gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Fri Feb 15 01:01:54 GMT+08:00 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-all.zip
```

Comment:
@melpulomas I am wondering if you changed this manually or was it your IDE? As I am not sure if we actually have to change this attribute. I have emailed the professor and awaiting her reply. I think may be if possible you can change this portion back to the default `distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-bin.zip` I think based on the -bin is a stripped down version for server usage and since it is the default in the project given I think we need not change it to -all. If I am not mistaken -all version is larger as it contains more files.

Reply:
@melpulomas I have clarified with our module coordinator, she stated we do not have to change the `distributionUrl`. I am wondering if there is a reason for the change if there are some changes you made which require it we can consider changing it else we try to keep it the same as the default which is -bin version